tests/foreman/ui_airgun/test_contentcredential.py

```diff
@@ -92,6 +92,33 @@ def test_positive_end_to_end(session, module_org, gpg_content):
         assert not session.contentcredential.search(new_name)
 
 
+@tier2
+def test_positive_search_scoped(session, gpg_content):
+    """Search for gpgkey by organization id parameter
+
+    :id: e1e04f68-5d4f-43f6-a9c1-b9f566fcbc92
+
+    :customerscenario: true
+
+    :expectedresults: correct gpg key is found
+
+    :BZ: 1259374
+
+    :CaseImportance: High
+    """
+    name = gen_string('alpha')
+    org = entities.Organization().create()
+    with session:
+        session.organization.select(org.name)
+        session.contentcredential.create({
+            'name': name,
+            'content_type': 'GPG Key',
```

Comment:
Think we need to create a constant for content_credentials types
```python
CONTENT_CREDENTIALS_TYPES = {
    'gpg': ''GPG Key'',
    'ssl': 'SSL Certificate'
}
```
and use it here

```python
 'content_type': CONTENT_CREDENTIALS_TYPES['gpg'], 

```